Lib/test/test_descr.py

```diff
@@ -4325,7 +4325,11 @@ class Evil(object):
             def __hash__(self):
                 return hash('attr')
             def __eq__(self, other):
-                del C.attr
+                try:
+                    del C.attr
+                except AttributeError:
```

Comment:
Although this fixes the failure, I think it may be making the test not testing what is supposed to check due to the fact that the __eq__ may be called twice or in a different situation that the test is expecting (check the bpo for more info).